DESCRIPTION

```diff
@@ -10,7 +10,7 @@ Authors@R:
 Description: What the package does (one paragraph).
 License: MIT
 Imports:
-  dplyr (>= 0.8.4), tibble (>= 2.1.3), tidyr (>= 1.0.2), MMWRweek (>= 0.1.1), lubridate (>= 1.7.4), purrr (>= 0.3.3), ggplot2, magrittr
+  dplyr (>= 0.8.4), tibble (>= 2.1.3), tidyr (>= 1.0.2), MMWRweek (>= 0.1.1), lubridate (>= 1.7.4), purrr (>= 0.3.3), ggplot2, magrittr, tidyverse
```

Comment:
if we are requiring tidyverse, then can we remove the other packages that are included in it? possible that all but MMWRweek and lubridate are included automatically?

Reply:
Or turning this around, could we import only the specific tidyverse packages we need?  (think we'd need to at least ad forcats)